analyzer/decrypt_crypto.cc

```diff
@@ -22,11 +23,13 @@ refactors as C++ development is not our main profession.
 // Import HILTI
 #include <hilti/rt/libhilti.h>
 
+namespace
+	{
```

Comment:
Not your fault, but our formatting looks clearly broken here. Either we should indent the whole namespace or none of it.